functions/__fisher_cache_list.fish

```diff
@@ -1,3 +1,3 @@
 function __fisher_cache_list
-    find -L $fisher_cache/* -maxdepth 0 -type d ^ /dev/null | sed 's|.*/||'
+    find -L $fisher_cache/* -printf '%f\n' -maxdepth 0 -type d ^ /dev/null
```

Comment:
Sorry, didn't saw this that day. Why it wouldn't work?

I guess you need to cherry-pick the `-h` unification commit out then.